pkg/webhook/policy.go

```diff
@@ -211,7 +220,8 @@ func (h *validationHandler) getDenyMessages(res []*rtypes.Result, req admission.
 		}
 	}
 	for _, r := range res {
-		if r.EnforcementAction == "deny" || r.EnforcementAction == "dryrun" {
+		enforcementAction = r.EnforcementAction
```

Comment:
`enforcementAction` needs to be set to the strictest observed enforcement action. Otherwise, a warning that occurs at the end of the result list with override any and all errors.

Reply:
s/errors/deny violations/g